### ISSUE TITLE
Pass config object to Cypress module API

### ISSUE THREAD

**Issue by bahmutov** [cypress-io/cypress#639]
Cypress 0.20.0 should accept config object via module api

* https://docs.cypress.io/guides/guides/command-line.html#Cypress-Module-API

Should be same as passing objects via command line switch cypress run --config pageLoadTimeout=100000,watchForFileChanges=false

```js
const cy = require('cypress')
cy.run({
  config: {
    pageLoadTimeout: 10000,
    watchForFileChanges: false
  }
})
```

**Comment by brian-mann** [comment]
The code for this is merged, but it has not been released yet.

**Comment by brian-mann** [comment]
Fixed in [`0.20.1`](https://docs.cypress.io/guides/references/changelog.html#0-20-1).
